lua/core/gitsigns.lua

```diff
@@ -2,6 +2,7 @@ local M = {}
 local Log = require "core.log"
 M.config = function()
   lvim.builtin.gitsigns = {
+    active = true,
```

Comment:
since we are passing `lvim.builtin.gitsigns` to `gitsigns.setup()` this causes the issue
<img width="464" alt="Screen Shot 2021-08-15 at 8 10 51 PM" src="https://user-images.githubusercontent.com/10992695/129484125-46ccd5b9-56bc-4c69-bf39-445669a58e58.png">

Reply:
Related: https://github.com/LunarVim/LunarVim/pull/1315#discussion_r689084719